app/javascript/modules/listingDetails/ListingDetailsImageCard.tsx

```diff
@@ -19,18 +20,25 @@ export const ListingDetailsImageCard = ({ listing }: ListingDetailsImageCardProp
   const shareButton = getAssetPath("share-button.svg")
   const shareButtonSelected = getAssetPath("share-button-selected.svg")
   const [shareImage, setShareImage] = useState(shareButton)
+  const listingImages: ImageItem[] = listing?.Listing_Images?.map((listingImage) => {
+    return {
+      url: listingImage.Image_URL,
+      description: listingImage.Image_Description,
+    }
+  })
 
   return (
     <header className="image-card--leader">
       <ImageCard
-        imageUrl={listing?.imageURL ?? fallbackImg}
-        href={`/listings/${listing.listingID}`}
+        imageUrl={(listingImages === undefined && listing?.imageURL) ?? fallbackImg} // Fallback if there is no listing images array.
+        images={listingImages?.length === 0 ? [{ url: fallbackImg }] : listingImages}
```

Comment:
Does the `fallbackImg` have a description that we would want to include?

Reply:
Talked with Bridget and T and decided that we should be hiding the fallback Image (and any descendants of the component) from a11y tools since the fallback is decorative and provides no functional value

Reply:
That makes sense. The only other consideration I can think of is having some helpful alt text describing that the fallback image is just a placeholder. May be helpful for low vision folks to make clear that the fallback image is not a photo representing the listing.

Reply:
Made a note to ask about this during our A11Y audit: https://sfgovdt.jira.com/browse/DAH-1641